test/mock/GovernorMetadataMock.sol

```diff
@@ -45,7 +45,7 @@ abstract contract GovernorMetadataMockBase is L2GovernorMetadata {
 }
 
 contract GovernorMetadataMock is GovernorMetadataMockBase, WormholeL2GovernorMetadata {
-  constructor(address _core) WormholeL2GovernorMetadata(_core, msg.sender) {
+  constructor(address _core) WormholeL2GovernorMetadata(_core, msg.sender, address(0x1b)) {
```

Comment:
We have a mock address in the Constants.sol ff the third parameter can be any address.

Reply:
Done in 165e503186e3748ec195858fcb3972d1021c0610